src/main/people.ts

```
@@ -46,7 +47,16 @@ function getPeoplePath(): string {
  * @returns the person's Basecamp id
  */
 export function getPersonId(personName: string): string | undefined {
-    const normalizedPersonName = normalizePersonName(personName);
+    const normalizedPersonName: string = normalizePersonName(personName);
+
+    // Pull the basecamp id from the member properties if possible
+    if(MEMBER_MAP.hasOwnProperty(normalizedPersonName) && MEMBER_MAP[normalizedPersonName].basecampId !== "") {
```

we want to log if the normalizedPersonName isn't found in MEMBER_MAP - that means they don't have an alias properly added

---

actually if they aren't found in the member map, i think it's ok to short circuit and end this entire execution and log the error, then `return undefined`. in an odd case where their name isn't a registered alias, but it matches with their basecamp name, i don't think we'd want that to keep silently passing. it's better to know upfront it's not a registered alias.